Sources/Array+Sweetness.swift

```diff
@@ -1,9 +0,0 @@
-import Foundation
```

Comment:
We could probably use a quick PR just removing this 👍

Reply:
I figured I should lump this in with a major version bump

Reply:
👍🏽